lib/pg_online_schema_change/orchestrate.rb

```diff
@@ -31,7 +31,7 @@ def setup_store
         # Set this early on to ensure their creation and cleanup (unexpected)
         # happens at all times. IOW, the calls from Store.get always return
         # the same value.
-        Store.set(:old_primary_table, "pgosc_op_table_#{client.table.downcase}")
+        Store.set(:old_primary_table, "pgosc_op_table_#{client.table.downcase}_#{pgosc_identifier}")
```

Comment:
Looks like the spec is failing due to the length. We would like to keep this length to 20 or maybe 21 characters, to not accidentally overflow in case the actual table name is longer and then result into production errors, since PG has a table name length limit. 

Do you want to see if you can repurpose the logic here to perhaps add an `index` (numeric). For example `pgosc_op_table_foobar_1`. 

Note: If `_1` already exists, then it should automatically increment the index from the previous value. 

It requires updating a few code paths and specs, so I am happy to take it over too. Let me know 🙌🏾